packages/@uppy/core/src/index.js

```diff
@@ -1706,11 +1698,7 @@ class Uppy {
   }
 }
 
-module.exports = function core (opts) {
-  return new Uppy(opts)
-}
-
 // Expose class constructor.
-module.exports.Uppy = Uppy
+module.exports = Uppy
```

Comment:
Why not both? When switching to ESM, people may prefer to use named import, it's also more convenient if they try to `export * from '@uppy/core'`.
```suggestion
module.exports = Uppy
module.exports.Uppy = Uppy
```

Reply:
Or only use named exports: https://basarat.gitbook.io/typescript/main-1/defaultisbad